expat/CMakeLists.txt

```diff
@@ -381,7 +381,7 @@ endforeach()
 #
 # C library
 #
-set(expat_SRCS
+set(expat_C_SOURCES
```

Comment:
Could you make it `_EXPAT_C_SOURCES` — all uppercase and with a leading underline?